### ISSUE TITLE
Move group list from /server-info to /update-groups endpoint

### ISSUE THREAD

**Issue by Derkades** [NamelessMC/Nameless#3378]
The global group list for the groups dropdown in StaffCP > API > Group sync is currently still sent as part of the /server-info request. This is confusing for users, as they need to enable group sync as well as the server data sender for group sync to work.

**Comment by samerton** [comment]
@Derkades do you recall if this is resolved by https://github.com/NamelessMC/Nameless/pull/3538 ?

**Comment by Derkades** [comment]
No, I don't remember that PR changing anything related to the Discord role list, so it is not resolved yet. But this is only a minor issue so it can be removed from the 2.2.0 milestone in my opinion
